treeherder/model/models.py

```diff
@@ -417,12 +417,11 @@ def cycle_data(self, repository, cycle_interval, chunk_size, sleep_time):
 
         jobs_cycled = 0
         while True:
-            jobs_chunk = list(self.filter(repository=repository, submit_time__lt=jobs_max_timestamp)
+            jobs_chunk = list(self.filter(submit_time__lt=jobs_max_timestamp)
                                   .order_by('id')
                                   .values_list('guid', flat=True)[:chunk_size])
```

Comment:
What is the runtime on this?  Since it does not use an index, I am concerned it will take too long.  Specifically, the very last chunk, which will scan the whole table looking for enough records to meet the `chunk_size` requirement.

* Can we wrap this in a timer and log it?  Then we can deploy it to see how it behaves.
* Can we wrap this in try/catch? Since I expect the last chunk will timeout.